src/main/java/com/exasol/adapter/document/files/stringfilter/wildcardexpression/renderer/regex/DirectoryAwareRegexRenderer.java

```diff
@@ -0,0 +1,21 @@
+package com.exasol.adapter.document.files.stringfilter.wildcardexpression.renderer.regex;
+
+import java.util.regex.Pattern;
+
+import com.exasol.adapter.document.files.stringfilter.wildcardexpression.WildcardExpression;
+
+/**
+ * This class renders a {@link WildcardExpression} into a regular expression that applies the non recursive wildcards
+ * not across directory borders.
```

Comment:
```suggestion
 * This class renders a {@link WildcardExpression} into a regular expression that applies the nonrecursive wildcards
 only for file / directory names, not for whole paths. It odes so by excluding the directory separator for the matches.
```

Reply:
Looks good. Please fix the typo on `It does so`.